symfit/core/fit.py

```diff
@@ -1832,6 +1721,7 @@ def __init__(self, model_dict, initial, *lsoda_args, **lsoda_kwargs):
             key=sort_func
         )
         self.independent_vars = sorted(set(d.variables[0] for d in model_dict), key=sort_func)
+        self.interdependent_vars = []  # Not yet supported for ODEModels
```

Comment:
Why? What's the problem?

Reply:
Mostly that I haven't thought about it yet ;). ODEModels are a pain in the ass.

I think actually that after this restructuring, ODEModels might be reinvented as some kind of  NumericalModel. Because that is mostly what they are. But more though on this is needed.

Reply:
Add a TODO or FIXME comment ;)

Reply:
By the way, turn ODEModels into NumericalModels in a different PR. Please. ;)